scraping_utils.py

```diff
@@ -0,0 +1,305 @@
+import sys
+import requests
+from bs4 import BeautifulSoup
+import urllib
+import pandas as pd
+import json
+from time import sleep, perf_counter as pf
+import regex as re
+import itertools as it
+import patterns as pt
+from course_info import Prereq
+
+#requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS += ':DES-CBC3-SHA'
+#timetable_url = 'https://web30.uottawa.ca/v3/SITS/timetable/Search.aspx'
+
+# Course Info Parameters
+course_url = 'https://catalogue.uottawa.ca/en/courses/'
+
+# Timetable Parameters
+with open("template_query.json", "r") as f:
+    old_form = json.load(f)
```

Comment:
Ahhh you're right, I forgot to do a `.copy()` down below.  I think I always intended to pass it as a parameter, but it just ended up as one of those things I did temporarily and forgot to go back and fix, but anyways, it's fixed for now.  

In the future, I do want to make it a class so it can implement a sort of framework that can hopefully be common across universities, but that's more long term.